ste/mgr-JobPartMgr.go

```diff
@@ -538,6 +547,13 @@ func (jpm *jobPartMgr) fileDstData(fullFilePath string, dataFileToXfer []byte) (
 	return azfile.FileHTTPHeaders{ContentType: jpm.inferContentType(fullFilePath, dataFileToXfer), ContentLanguage: jpm.fileHTTPHeaders.ContentLanguage, ContentEncoding: jpm.fileHTTPHeaders.ContentEncoding, ContentDisposition: jpm.fileHTTPHeaders.ContentDisposition, CacheControl: jpm.fileHTTPHeaders.CacheControl}, jpm.fileMetadata
 }
 
+func (jpm *jobPartMgr) bfsDstData(fullFilePath string, dataFileToXfer []byte) (headers azbfs.BlobFSFileHTTPHeaders) {
+	if jpm.planMMF.Plan().DstBlobData.NoGuessMimeType || dataFileToXfer == nil {
+		return jpm.bfsFileHTTPHeaders
+	} //TODO: come back!
```

Comment:
What's this comment for?

Reply:
I was going to come back there if something I did elsewhere didn't manage to get HTTP headers through for BlobFS. It seems I have a strong tendency to leave debug info leftover. Should probably be more careful about that and use safer practices to keep track of what I'm doing.